cmd/krel/cmd/push.go

```diff
@@ -152,51 +161,56 @@ func init() {
 }
 
 func runPushBuild(opts *pushBuildOptions) error {
-	log.Println("unimplemented")
-
 	var latest string
 	releaseKind := opts.releaseKind
 
 	// Check if latest build uses bazel
 	dir, err := os.Getwd()
 	if err != nil {
-		log.Fatal(err)
+		logrus.Errorf("Unable to get working directory: %v", err)
```

Comment:
I think we can `errors.Wrapf()` here and return the errors. This will result in a fatal log. Now it seems to me that we log them twice, right?